### PR TITLE
docs: small cleanup

### PR DESCRIPTION
**Pull request by artofcomputing**
### SUMMARY

This PR performs some small cleanup fixes in the documentation with the intent of following #27612 discussion.

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

**Comment by mistercrunch** [comment]
`pre-commit` seems to be complaining. You might want to try `pre-commit run --all-files`  also `pre-commit install` might prevent this and save some cycles.

File: docs/docs/installation/installing-superset-using-docker-compose.mdx [hunk 1]
```diff
@@ -10,8 +10,8 @@ version: 1
 **DO NOT USE THIS FOR PRODUCTION!**
 
 The fastest way to try Superset locally is using Docker Compose on a Linux or Mac OSX
-computer. Superset does not have official support for Windows, so we have provided a VM workaround
-below. It's also the easiest way to launch a fully functioning **development environment** quickly.
+computer. Superset does not have official support for Windows. It's also the easiest
```
Comment (rusackas) [inline docs/docs/installation/installing-superset-using-docker-compose.mdx:13]:
```suggestion
computer. Superset does not have official support for Windows. It's also the easiest
```

**Comment by mistercrunch** [comment]
This will need a rebase to trigger the required checks. I wish there was a button for that on the GH UI

**Comment by artofcomputing** [comment]
>pre-commit seems to be complaining. You might want to try pre-commit run --all-files also pre-commit install might prevent this and save some cycles.

Ah thanks for reminding me!

**Pull request merged**